File: condicoes/1040.py
#1040 MÉDIA 3
valores = input().split(" ")
nota = [float(valor) for valor in valores]

n1 = nota[0]*2
n2 = nota[1]*3
n3 = nota[2]*4
n4 = nota[3]*1
media = (n1+n2+n3+n4)/10
print("Media: {:.1f}" .format(media))

if media >= 7.0:
    print("Aluno aprovado.")
elif media < 5.0:
    print("Aluno reprovado.")
else:
    print("Aluno em exame.")

    notaDoExame = float(input("DIGITE A NOTA DO EXAME: "))
    print("Nota do exame:",notaDoExame)
    mediaFinal = (notaDoExame+media)/2

    if mediaFinal >= 5.0:
        print("Aluno aprovado.")
        print("Media final: {:.1f}" .format(mediaFinal))
    else:
        print("Aluno reprovado.")
        print("Media final: {:.1f}" .format(mediaFinal))

File: javascript/1040.js
/*Média 3*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let nota = input.split("\n");
let linhas = nota.shift();
let lines = linhas.split(" ");

let N1 = parseFloat(lines.shift());
let N2 = parseFloat(lines.shift());
let N3 = parseFloat(lines.shift());
let N4 = parseFloat(lines.shift());

let Media = ((N1 * 2) + (N2 * 3) + (N3 * 4) + (N4 * 1)) /10;
console.log("Media: " + Media.toFixed(1));

if(Media >= 7.0){
  console.log("Aluno aprovado.");
}else if(Media < 5.0){
  console.log("Aluno reprovado.");  
}else{
  console.log("Aluno em exame.");
  let Exame = parseFloat(nota.shift());
  console.log("Nota do exame: " + Exame.toFixed(1));
  let Final = (Exame + Media) /2;

if(Final >= 5.0){
  console.log("Aluno aprovado.");
  console.log("Media final: " + Final.toFixed(1));
}else{
  console.log("Aluno reprovado.");
  console.log("Media final: " + Final.toFixed(1));
}
}